hs_communities/models.py

```diff
@@ -0,0 +1,43 @@
+from django.db import models
+
+
+class Topic(models.Model):
+    name = models.CharField(editable=True, null=False, max_length=255)
+
+    def __str__(self):
+        return "{}".format(self.name)
+
+
+class TopicEntry(models.Model):
+    topic = models.ForeignKey(Topic,
+                              null=False,
+                              editable=True,
+                              help_text='One topic entry for a resource')
+    order = models.IntegerField(null=False,
```

Comment:
not sure how I feel about order... it's effectively just the generated id.  There isn't a way to reorder outside of writing more code and I'm unsure if there is any intention to reorder.  Also what happens if the two entries have the same order?  Would that also throw off the code that generates order for a new topic by taking computing max +1?

Reply:
I think if we're not using it, it should be pulled until we have a use for it.  It appears to be not completely thought out and will likely need to be reworked a fair amount anyways (mechanisms for adding/reordering, handling duplicate order number)

Reply:
Addressed in 5d9b4648eb93c6c129b8ae142ade2e40bcb8853b

Reply:
https://github.com/hydroshare/hydroshare/commit/5d9b4648eb93c6c129b8ae142ade2e40bcb8853b#commitcomment-34936505

Reply:
The Model was re-written can you confirm your concerns are addressed?

Reply:
There is nothing in that commit or your response that addresses the order field and that the only ordering on topics we're currently using is by name.